app/Jobs/FetchCollectionBannerBatch.php

```diff
@@ -44,13 +45,29 @@ public function handle(): void
         $metadata = Alchemy::getContractMetadataBatch($this->collectionAddresses, $this->network);
 
         $metadata->each(function (Web3ContractMetadata $data) {
+            // Skip this iteration because bannerImageUrl is null
+            if (is_null($data->bannerImageUrl)) {
+                return false;
+            }
+
             Log::info('Updating collection banner', [
                 'collection_address' => $data->contractAddress,
                 'banner' => $data->bannerImageUrl,
             ]);
 
-            Collection::query()->where('address', $data->contractAddress)
-                ->update(['extra_attributes->banner' => $data->bannerImageUrl]);
+            Collection::query()
+                ->where('address', $data->contractAddress)
+                ->update([
+                        'extra_attributes' => DB::raw("
```

Comment:
@shahin-hq as discussed, should be double checked

Reply:
Checked, but couldn't make it work with two column updates, that is why changed the implementation. Please check.